Geometry_Engine/Compute/CurveProximity.cs

```diff
@@ -1034,22 +1034,6 @@ public static partial class Compute
             return result;
         }
 
-        /***************************************************/
-
-        [NotImplemented]
-        public static Output<Point, Point> CurveProximity(this ICurve curve1, Ellipse curve2, double tolerance = Tolerance.Distance)
```

Comment:
Are we removing this in favour of an interface method or just removing it completely?

Reply:
Sorry - just found it - agreed.